src/sdl2/mixer/mod.rs

```diff
@@ -818,45 +834,66 @@ impl<'a> fmt::Debug for Music<'a> {
 }
 
 impl<'a> Music<'a> {
-    /// Load music file to use.
-    pub fn from_file<P: AsRef<Path>>(path: P) -> Result<Music<'static>, String> {
-        let raw = unsafe {
-            let c_path = CString::new(path.as_ref().to_str().unwrap()).unwrap();
-            mixer::Mix_LoadMUS(c_path.as_ptr())
-        };
+    fn load_bytes(
+        buf: *const u8,
+        len: usize,
+        owned: Option<NonNull<[u8]>>,
+    ) -> Result<Music<'static>, String> {
+        let rw = unsafe { sys::SDL_RWFromConstMem(buf as *const c_void, len as c_int) };
+        if rw.is_null() {
+            return Err(get_error());
+        }
+        let raw = unsafe { mixer::Mix_LoadMUS_RW(rw, 0) };
         if raw.is_null() {
             Err(get_error())
         } else {
             Ok(Music {
-                raw,
-                owned: true,
+                raw: raw,
+                owned_data: owned,
                 _marker: PhantomData,
             })
         }
     }
 
     /// Load music from a static byte buffer.
-    #[doc(alias = "SDL_RWFromConstMem")]
```

Comment:
I see you moved this alias to `from_owned_bytes`, but it can stay here too, right?